soc/arm/atmel_sam0/common/power_samd2x.c

```diff
@@ -0,0 +1,100 @@
+/*
```

Comment:
This commit seems unrelated to the RTC fix.  Please split it out.

Reply:
These changes are related and required because a special synchronization is required when coming out of the STANDBY sleep state.  Specifically, _sys_pm_power_state_exit_post_ops() provides a busy wait until the RTC starts ticking again.